django/forms/widgets.py

```diff
@@ -511,11 +517,17 @@ def value_from_datadict(self, data, files, name):
         return bool(value)
 
 
-class Select(Widget):
+class ChoiceWidget(Widget):
     allow_multiple_selected = False
+    input_type = None
+    template_name = None
+    option_template_name = None
+    add_id_index = True
+    checked_attribute = {'checked': 'checked'}
```

Comment:
To preserve the 1.11 move to HTML5 boolean syntax I believe this line should be changed to `checked_attribute = {'checked': True}`.

Same for the `Select` class on line 690: `checked_attribute = {'selected': True}`